face_track/face_track.py

```diff
@@ -144,8 +147,20 @@ def __init__(self):
 
 		# Which face to look at
 		# rospy.Subscriber(self.TOPIC_FACE_TARGET, xxxFaceEvent, xxxself.face_event_cb)
-		rospy.Subscriber("perceived_text", String,
-			self.user_said_cb)
+		#rospy.Subscriber("perceived_text", String,
+		#	self.user_said_cb)
```

Comment:
can you plese stop commenting out code!  If the code is dead, then please remove it, instead of commenting it out.

If you are not done with development, please don't submit a pull request.